css/styles.css

```diff
@@ -6,6 +6,11 @@ body {
   background-color: #e5e5e5;
 }
 
+main {
+  max-width: 1263px;
```

Comment:
- [OPTIONAL] Great job on implementing the Figma design for the desktop version perfectly well, 👍 💯 ... 
- I notice that on a very big devices, the `main` section of your page does not span fully to the left and right of the screen and this is due to the specific max width you gave, Please I'll suggest you remove this, Your portfolio looks great without it on all screens. Thanks 😄 
- SCREENSHOT FOR REFERENCE:
![Screenshot 2023-03-30 191242](https://user-images.githubusercontent.com/107632780/228928608-69120ed2-f71b-4294-b287-11301c179f23.png)